xml/System.Diagnostics/TagList.xml

```diff
@@ -454,9 +476,10 @@
         <Parameter Name="item" Type="System.Collections.Generic.KeyValuePair&lt;System.String,System.Object&gt;" />
       </Parameters>
       <Docs>
-        <param name="item">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="item">The tag to remove from the <see cref="T:System.Diagnostics.TagList" />.</param>
+        <summary>Removes the first occurrence of a specific object from the <see cref="T:System.Diagnostics.TagList" />.</summary>
+        <returns>
+          <see langword="true" /> if item is successfully removed; otherwise, <see langword="false" />. This method also returns <see langword="false" /> if item was not found in the <see cref="T:System.Diagnostics.TagList" />.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if <paramref name="item" /> is successfully removed; otherwise, <see langword="false" />. This method also returns <see langword="false" /> if <paramref name="item" /> was not found in the <see cref="T:System.Diagnostics.TagList" />.</returns>
```